alchemist/alchemist-full/build.gradle.kts

```diff
@@ -13,3 +13,11 @@ dependencies {
     }
     testImplementation(rootProject)
 }
+
+plugins {
+    application
+}
+
+application {
+    mainClassName = "it.unibo.alchemist.Alchemist"
+}
```

Comment:
Why this change?

Reply:
I could not get Alchemist to run correctly otherwise (classpath/subproject issues).
With this change I can do `./gradlew run --args="-y foo.yaml"`

Should I revert this?

Reply:
Uhmm... what do you mean? You should be able to run by launching with the alchemist-full classpath, and you should be able to run using the fatJar as well.

I'm thinking about this change. It's not wrong per se, but I'm unsure if it taints the build.

Reply:
My IDEA has some issues running Alchemist, ever since the structure of the launchers got refactored - I can't get it to find all the launchers across the entire classpath (even if I run from `alchemist-full`).

Now I tried running with the runShadow task, like this:
`./gradlew runShadow --args="-y /home/vuksa/Documents/Alchemist/alchemist/alchemist-incarnation-protelis/build/resources/test/testbase.yml -fxui"`

But I get this exception:
`java.util.MissingResourceException: The string resource '_UI_DiagnosticRoot_diagnostic' could not be located`

And [this](https://pastebin.com/j0vKct1K) is the full output.

The `run` task I added executes correctly with the same args though.